pkg/internal/testhelpers/postgres_container.go

```diff
@@ -123,16 +123,18 @@ func (e ExtensionState) GetPGMajor() string {
 
 func (e ExtensionState) GetDockerImageName() (string, error) {
 	var image string
-	PGMajor := e.GetPGMajor()
-	PGTag := "pg" + PGMajor
+	//PGMajor := e.GetPGMajor()
```

Comment:
I would like to express that I'm working on changes which should rework this whole thing completely, so I'm fine with merging this as-is, assuming that we get the other stuff merged soon after.